### ISSUE TITLE
Invalid or unsupported zip format. No END header found

### ISSUE THREAD

**Issue by anemol** [NodeBB-Community/nodebb-plugin-emoji-one#8]
Linux server Ubuntu 16.04 (64bits.)
Nodebb 1.4.5 and 1.5

Lasted versions of plugins installed.

Installed Sets > Upload files

Log:
```
17/5 23:46:10 [24043] - error: Invalid or unsupported zip format. No END header found
17/5 23:46:10 [24043] - info: [app] Shutdown (SIGTERM/SIGINT) Initialised.
17/5 23:46:10 [24043] - info: [app] Database connection closed.
17/5 23:46:10 [24043] - info: [app] Web server closed to connections.
17/5 23:46:10 [24043] - info: [app] Shutdown complete.
[cluster] Child Process (24043) has exited (code: 1, signal: null)
[cluster] Spinning up another process...

```

Tried to download manually this file http://emojione.com/wp-content/uploads/assets/e1-svg.zip but failed. File is not zip or corrupted.

**Comment by ghost** [comment]
Fixed within #7 
Thanks for report!

**Comment by anemol** [comment]
How to update with the new version (1.2)?

NodeBB doesn't offer any update for the plugin (maybe the npm module is not updated?)

`Installed 1.1.5 | Latest 1.1.5`


I see that the npm module version is 1.2.1

But if i try to install manually:
```
npm install nodebb-plugin-emoji-extended nodebb-plugin-emoji-one
npm WARN deprecated wrench@1.5.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
nodebb@1.4.6 /home/fofo/nodebb
├── nodebb-plugin-emoji-extended@1.1.1 
└─┬ nodebb-plugin-emoji-one@1.1.5 
  ├── adm-zip@0.4.7 
  └── wrench@1.5.9 


```
